cluster-autoscaler/cloudprovider/linode/README.md

```diff
@@ -0,0 +1,46 @@
+# Cluster Autoscaler for Linode
+
+The cluster autoscaler for Linode scales nodes in a LKE cluster.
+
+A cluster autoscaler node group is composed of multiple LKE Node Pools of the same Linode type (e.g. g6-standard-1, g6-standard-2), each holding a *single* linode.
+
+At every scan interval the cluster autoscaler review every LKE Node Pool and if:
+* it is not among the the ones to be excluded as defined in the configuration file;
+* it holds a single linode;
+
+then it becomes part of the node group holding LKE Node Pools of the linode type it has, or a node group is created with this LKE Node Pool inside if there are no node groups with that linode type at the moment.
+
+Scaling is achieved adding LKE Node Pools to node groups, *not* increasing the size of a LKE Node Pool, that must stay 1. The reason behind this is that Linode does not provide a way to selectively delete a linode from a  LKE Node Pool and decrease the size of the pool with it.
+
+## Configuration
+
+The cluster autoscaler automatically select every LKE Node Pool that is part of a LKE cluster, so there is no need define the `node-group-auto-discovery`  or `nodes` flags, see [examples/cluster-autoscaler-autodiscover.yaml](examples/cluster-autoscaler-autodiscover.yaml) for an example of a kubernetes deployment.
```

Comment:
I've updated the README with more information on the reasons behind the choices. At the moment I feel using the LKE cluster tags is no better choice than the INI file, and we cannot use the tags in a node as you explained because nodes are disposable.